doc/src/Howto_bpm.rst

```diff
@@ -107,7 +107,11 @@ bond lists is expensive.  By setting the lj weight for 1-2 bonds to
 zero, this turns off pairwise interactions.  Even though there are no
 charges in BPM models, setting a nonzero coul weight for 1-2 bonds
 ensures all bonded neighbors are still included in the neighbor list
-in case bonds break between neighbor list builds.
+in case bonds break between neighbor list builds. If bond breakage is
+disabled during a simulation run by setting the *break* keyword to *no*,
+a zero coul weight for 1-2 bonds can be used to exclude bonded atoms
```

Comment:
What about the LJ weights ?  What should they be set to ?

Reply:
- If `overlay/pair yes` is used, the lj weights **must be** 1 X Y, regardless of `break yes` or `break no`.

The code currently only checks for LJ[1]==1. It does not enforce X=1 or Y=1, nor coul weights to be 1 1 1.
The main downside of not enforcing LJ[2]=LJ[3]=Coul[1]=Coul[2]=Coul[3]=1 would be the cost of building the 1-3 and 1-4 special bonds if a user selects a value <1. We could enforce that, or leave it as is, please let me know your thoughts @jtclemm @sjplimp 

- If `overlay/pair no` is used, the lj weights **must be** 0 1 1, regardless of `break yes` or `break no`

Let me rephrase the doc to explicitly include `special_bonds lj 0 1 1 coul 0 1 1` per @akohlmey's comment about `coul` resetting `lj`.

Reply:
> We could enforce that, or leave it as is, please let me know your thoughts @jtclemm @sjplimp

Oh right, no limits were placed on the `overlay yes` option. I don't have strong opinions, but I would lean on the side of simply always requiring the LJ[2] = LJ[3] = Coul[2] = Coul[3] = 1.0. The BPM package will already error if used with 3- or 4-body interactions, so this shouldn't add any practical restrictions.

I do have a version of 3-body BPM interactions which I may eventually update and add to the package (and could extend it to 4-body), but I think it's best to worry about those cases later.

Reply:
I think @jtclemm 's comment is the right approach.  Have the user follow the simple rule now - if it needs to be relaxed for a later extension, do that later.

Reply:
Sounds good to me. I updated the error check to enforce LJ[1] = LJ[2] = LJ[3] = Coul[1] = Coul[2] = Coul[3] = 1.0 when `overlay/pair yes` is used and updated the documentation accordingly.

Reply:
Thanks for adding those checks Jibril. I think the PR should be good to go.